js/app2.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+class Book {
+  #list
+
+  constructor() {
+    const localList = JSON.parse(localStorage.getItem('stuff'));
+    if (localList !== null) {
+      this.#list = localList.arr;
+    }
+  }
+
+  #g(j) {
+    return this.document.getElementById(j);
+  }
+
+  #stack(templateString) {
+    const template = this.document.createElement('template');
+    template.innerHTML = templateString.trim();
+    document.getElementById('books').appendChild(template.content.firstChild);
+  }
+
+  updateList() {
+    document.getElementById('books').innerHTML = '';
+    for (let j = 0; j < this.#list.length; j++) {
```

Comment:
* This line throws an error because  you are calling the `.length` method on `#list` which has no value, kindly fix it 🙏